products/opensuse/main.pm

```diff
@@ -445,6 +445,7 @@ sub load_consoletests {
         loadtest "console/http_srv";
         loadtest "console/mysql_srv";
         loadtest "console/dns_srv";
+        loadtest "console/dstat";
```

Comment:
the "console" tests are the one that should run in most scenarios for the more critical system features. It would be better to move "dstat" to the so called "extra tests", see the function `load_extra_tests` in `lib/main_common.pm`